docs/config_reference.rst

```diff
@@ -223,6 +223,7 @@ System Partition Configuration
    Supported schedulers are the following:
 
    - ``local``: Jobs will be launched locally without using any job scheduler.
+   - ``flux``: Jobs will be launched using the `Flux Framework <https://flux-framework.org/>`_ scheduler.
```

Comment:
Yes when you install flux the Python bindings are installed - there was a setting that you could build without them but actually my PR to remove this entirely (because Flux needs Python) was merged this morning! So if they have flux available to use, they will have the Python bindings, generally speaking. The admin that sets up flux would have to go out of their way to make them not available, which (in my developer experience using flux) I haven't seen yet. So I think we don't need to put that comment, at least for the time being until it's a proven issue.